airbyte-integrations/connectors/source-xero/source_xero/source.py

```diff
@@ -83,8 +83,8 @@ def get_authenticator(config: Mapping[str, Any]) -> Mapping[str, Any]:
         return XeroSingleUseRefreshTokenOauth2Authenticator(
```

Comment:
It seems odd that we have a custom component for that. It seems to be related to HTTP requests we are performing. Eventually, it might be interesting to increase the capabilities of OAuth in the CDK to avoid having those custom things